Asgardia.Constitution.Proposal.txt

```diff
@@ -242,32 +243,32 @@ Article 21. Security Areas
 Article 22. Security of Citizens
 § 1. Asgardia defines the security of citizens as the ability of the state and the citizen to overcome threats and minimise risks in the present and future.
 § 2. Citizen security is ensured in Asgardian territory.
-§ 3. Asgardia guarantees protection from any slavery or servitude, physical or emotional violence and compulsion, unlawful dependence and restrictions, and unlawful obligations or subordination, within its territory. 
-§ 4. State bodies and diplomatic representative bodies of Asgardia in Earth nations must use all peaceful means and measures to assist the provision of physical security of its citizens outside of Asgardia in accordance with Asgardia’s Constitution and laws, generally recognised international principles and norms, international treaties, laws and rules of nations in which Asgardian citizens reside.
+§ 3. Asgardia guarantees protection from any slavery or servitude, physical or emotional violence and compulsion, unlawful dependence and restrictions, and unlawful obligations or subordination within its territory. 
+§ 4. State bodies and diplomatic representative bodies of Asgardia in Earth nations must use all peaceful means and measures to assist the provision of physical security of its citizens outside of Asgardia in accordance with Asgardia’s Constitution and laws, generally recognised international principles and norms, international treaties, and rules of nation states in which Asgardian citizens reside.

```

Comment:
If this is what was originally intended, laws is already captured in the first item of the list. If you meant rules and laws of nation states, I recommend changing to “… and the rules and laws of nation states…”